tests/integration/test_application.py

```diff
@@ -211,10 +211,7 @@ def test_run_always_load_project(monkeypatch, app, cmd):
     monkeypatch.setenv("CRAFT_DEBUG", "1")
     monkeypatch.setattr("sys.argv", ["testcraft", cmd])
 
-    with pytest.raises(FileNotFoundError) as raised:
-        app.run()
-
-    assert str(raised.value).endswith("/testcraft.yaml'") is True
```

Comment:
I think we should inspect the log/emitter here to make sure the error message is still useful